### ISSUE TITLE
Dataloader object list

### ISSUE THREAD

**Issue by jyzhang2333** [gaobb/AdaptCLIP#11]
Hi, impressed work and Thanks for sharing the code~

I have a question regarding the dataloader in the Dataset class, it seems the objects list is from test set by default, if I want to use the train mode, should I change it to training objects list? 

<img width="1225" height="586" alt="Image" src="https://github.com/user-attachments/assets/3b954b5d-3381-49ea-93d9-3f04a60e05b9" />
